### PR TITLE
Add README jump hyperlink

### PR DESCRIPTION
**Pull request by TianTianBigWang**
Signed-off-by: TianTianBigWang <zilong.wang@daocloud.io>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 <img src="./docs/images/kubeedge-logo-only.png">
 
+[![README](https://img.shields.io/badge/README-%E4%B8%AD%E6%96%87-brightgreen)](./README_zh.md)
```
Comment (gy95) [inline README.md:10]: seems need to be moved to be before the kubeedge logo, put it together with other like `build status` and `releases`    and this img should be upload to the kubeedge path  like https://img.shields.io/github/license/kubeedge/kubeedge.svg?style=flat-square

Comment (MoYuanDan) [reply README.md]: ok

Comment (TianTianBigWang) [reply README.md]: ok

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 [![LICENSE](https://img.shields.io/github/license/kubeedge/kubeedge.svg?style=flat-square)](/LICENSE)
 [![Releases](https://img.shields.io/github/release/kubeedge/kubeedge/all.svg?style=flat-square)](https://github.com/kubeedge/kubeedge/releases)
 [![Documentation Status](https://readthedocs.org/projects/kubeedge/badge/?version=latest)](https://kubeedge.readthedocs.io/en/latest/?badge=latest)
+[![README_zh](https://img.shields.io/badge/README-%E4%B8%AD%E6%96%87-brightgreen.svg?style=flat-square)](./README_zh.md)
```
Comment (fisherxu) [inline README.md:7]: How about add the link directly in a new line? eg `English | [简体中文](./README-zh.md)` , users can find it more easily than badge.

Comment (TianTianBigWang) [reply README.md:7]: This can also be, I just think the use of icons look good

Comment (fisherxu) [reply README.md:7]: Will there be users looking into the badge/icons, I usually ignore them ;-)

Comment (TianTianBigWang) [reply README.md:7]: Or put this icon on a new line, so that it will be noticed by the user as the first icon.😁

Comment (gy95) [reply README.md:7]: IMO, this format `English | [简体中文](./README-zh.md)` in a new line is better, users can click and jump directly to the zh README.

Comment (TianTianBigWang) [reply README.md]: done

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**